src/content/developers/docs/nodes-and-clients/nodes-as-a-service/index.md

```diff
@@ -44,17 +44,18 @@ Read more about the [benefits of running your own node](/developers/docs/nodes-a
 
 Here is a list of some of the most popular Ethereum node providers, feel free to add any that are missing! Each node service offers different benefits and features in addition to free or paid tiers, you should investigate which ones best suit your needs prior to making a decision.
 
-- [**Alchemy**](https://www.alchemy.com/)
+- [**Alchemy**](https://alchemy.com/?a=ethereum-org)
   - [Docs](https://docs.alchemyapi.io/)
   - Features
-    - Free tier option
-    - Scale as you go
-    - Free archival data
-    - Analytics tools
-    - Dashboard
-    - Unique API endpoints
-    - Webhooks
-    - Direct support
+    - Largest free tier with 300M compute units per month (~30M getLatestBlock requests)
+    - Multichain support for Solana, Polygon, Starknet, Optimism, Arbitrum, Flow, Crypto.org
```

Comment:
Let's list Ethereum L2s or at least Ethereum aligned chains here. IMO Alchemy supporting Solana isn't really relevant to users of Ethereum.

```suggestion
    - Multichain support for Polygon, Starknet, Optimism, Arbitrum
```